internal/cmd/run_test.go

```diff
@@ -15,52 +15,180 @@
 package cmd
 
 import (
+	"bytes"
 	"context"
 	"errors"
+	"fmt"
 	"os"
 	"path/filepath"
 	"testing"
 
+	"github.com/urfave/cli/v2"
+
+	"github.com/tetratelabs/func-e/internal/moreos"
+
 	"github.com/stretchr/testify/require"
 
 	"github.com/tetratelabs/func-e/internal/globals"
 
 	"github.com/tetratelabs/func-e/internal/version"
 )
```

Comment:
I just realized that this messy import statement order hasn't been detected?

Reply:
https://github.com/tetratelabs/func-e/pull/350/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L139-L142 is the culprit 🗡️

Reply:
wonder if there's a cleaner way to accomplish that. meanwhile I can put the awk back in and say this groups imports neatly. /me grumbles why GOIMPORTS can't just format properly

Reply:
yeah I also wonder and would like to have better ways 😄

Reply:
ok I've restored the logic (in new style) PTAL!